src/SmartComponents/Reports/Reports.js

```diff
@@ -11,7 +11,7 @@ import useFeature from 'Utilities/hooks/useFeature';
 
 const QUERY = gql`
 {
-    profiles(search: "has_test_results = true", limit: 1000){
+    profiles(search: "(external = false AND has_policy_test_results = true) OR (has_policy = false AND has_test_results = true)", limit: 1000){
```

Comment:
Do you guys know how to break the line?

Reply:
We should probably define a constant for it at this point :)

Reply:
This (or similar) seems to work:

```suggestion
    profiles(search: "(external = false AND has_policy_test_results = true)
                      OR (has_policy = false AND has_test_results = true)",
                      limit: 1000){

```